Undo a patch from 5.17.9 which makes one of my systems crash
Patch by Martin Vaeth <martin at mvath.de> from May 29, 2022
--- 1/drivers/video/fbdev/vesafb.c
+++ 1/drivers/video/fbdev/vesafb.c
@@ -192,8 +192,6 @@
 	if (info->screen_base)
 		iounmap(info->screen_base);
 	release_mem_region(info->apertures->ranges[0].base, info->apertures->ranges[0].size);
-
-	framebuffer_release(info);
 }
 
 static struct fb_ops vesafb_ops = {
@@ -494,6 +492,7 @@
 	unregister_framebuffer(info);
 	if (((struct vesafb_par *)(info->par))->region)
 		release_region(0x3c0, 32);
+	framebuffer_release(info);
 
 	return 0;
 }
